internal/http/routes/api/v1/auth.go

```diff
@@ -150,6 +163,34 @@ func (r *AuthAPIRoutes) meHandler(c *gin.Context) {
 	response.Send(c, http.StatusOK, ctx.GetAccount())
 }
 
+func (r *AuthAPIRoutes) settingsHandler(c *gin.Context) {
```

Comment:
Remember to add the swagger tags to the handler so it will automatically show up in the [swagger documentation](https://github.com/go-shiori/shiori/blob/master/docs/APIv1.md).